test/e2e/storage/drivers/csi.go

```diff
@@ -173,7 +178,12 @@ func (g *gcePDCSIDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
 	cs := f.ClientSet
 	config := g.driverInfo.Config
 	framework.SkipUnlessProviderIs("gce", "gke")
-	framework.SkipIfMultizone(cs)
+	if !g.driverInfo.Config.AllAlpha {
```

Comment:
is `!AllAlpha` the right signal to be skipping on? it seems more like a related signal but not the actual "reason" to skip the test

Reply:
Whether we skip depends on whether or not topology is enabled, which depends on `AllAlpha`

Reply:
I guess i'm wondering if there is a "topology enabled" signal we can look for directly, that way we wont have to change this part of the test when toplogy is no longer alpha

Reply:
Can we define some sort of Topology capability and skip tests based off of that?  Similar to plugin capabilities I'm adding here: https://github.com/kubernetes/kubernetes/pull/70756

Reply:
Other drivers that implement topology will probably want this, like ebs.

Reply:
When this moves to beta, we can delete this AllAlpha flag and check entirely, and always run all tests with delayed binding (although this may uncover some problems with test cases that were not written with delayed binding in mind)

Reply:
Yup I agree the check can be removed after topology is beta, although don't we still want to have tests with immediate binding for full coverage?

Either way once the feature is beta, both immediate and delayed binding will work in both single zone and multizone settings